cvat-canvas3d/src/typescript/canvas3d.ts

```diff
@@ -17,6 +19,9 @@ interface Canvas3d {
     mode(): Mode;
     render(): void;
     keyControls(keys: KeyboardEvent): void;
+    mouseControls(type: string, event: MouseEvent): void;
```

Comment:
``type`` should be ``enum``, not just string because we have a set of possible values

Reply:
Resolved

Reply:
@manasars I still see simple ``string`` here